### PR TITLE
Add a way to launch the map editor with the same desktop file

### PR DESCRIPTION
**Pull request by sevu**
similar to the extra options which Firefox has
with an right click one can launch the editor now
one could as well add an "cheating mode" option for -d
Maybe to some Developers an desktop action for the test scenario is useful too?
(just for inspiration, not to be added to the PR)

Many distributions use modified desktop files, and add a 2nd
to launch the map editor. I think that's a better idea, because
it's less hidden than this feature. However, I think one doesn't
out-rule the other.

The translations are taken from downstream's desktop files for
the map editor - unfortunately they aren't complete

Basically I think this can go in as is.

actions

### CODE REVIEW

**Comment by CelticMinstrel** [comment]
I think an option for the test scenario is pretty useless (pretty sure only Wesnoth developers would ever need that), but an option for the MicroAI test scenario could be useful, as it's intended more as a demo for add-on developers (the command-line option would be `-tmicro_ai_test`, I think).
